test/dataExtension.test.js

```diff
@@ -29,9 +29,9 @@ describe('dataExtension', () => {
                 'returned metadata was not equal expected'
             );
             assert.equal(
-                Object.values(testUtils.getAPIHistory()).flat().length,
-                6,
-                'Unexpected number of requests made'
+                testUtils.getAPIHistoryLength(),
+                5,
+                'Unexpected number of requests made. Run testUtils.getAPIHistoryDebug() to see the requests'
```

Comment:
how does one "run" this?

Reply:
by adding testUtils.getAPIHistoryDebug() to your testing code?
I didn't want that to be in the code in some static fashion because in most situations it's clear what went wrong. But if it's not, then we now have that extra method for more debugging.

Reply:
on your workaround question: well, we are calling the method provided by the external library but it doesn't seem to work. It caches the token despite that. From my POV the issue needs to be resolved by the lib owner eventually. Until then, I think this is a perfectly fine solution for the issue